dev/bots/analyze.dart

```diff
@@ -1313,38 +1313,35 @@ Future<void> verifyRepositoryLinks(String workingDirectory) async {
     'tpn/winsdk-10',
   };
 
-  const List<String> linkPrefixes = <String>[
-    'https://raw.githubusercontent.com/',
-    'https://github.com/',
-  ];
+  // See bad_repository_links.dart for examples of repository links that are not allowed.
```

Comment:
Just reading this line it's not obvious where `bad_repository_links.dart `

Would to be reasonable to mention the folliowing path and wrap the comment in two lines? 
`dev/bots/test/analyze-test-input/root/packages/foo/bad_repository_links.dart`

Reply:
Sure!